app/views/coronavirus_form/cookies.html.erb

```diff
@@ -38,36 +38,31 @@
     <% end %>
   </div>
 
-  <div class="cookie-settings__form-wrapper">
-    <p>
-      We use <%= t('cookies.settings_page.cookies').count %> types of cookie.
-      You can choose which cookies you're happy for us to use.
-    </p>
+  <% t('cookies.settings_page.tables').map do |table| %>
+    <div class="govuk-body govuk-!-padding-top-6">
+      <h2><%= sanitize(table.dig(:header)) %></h2>
+      <p><%= sanitize(table.dig(:text_html)) %></p>
+      <%= render "govuk_publishing_components/components/table", {
+        head: [
+          { text: "Name" },
+          { text: "Purpose" },
+          { text: "Expires" },
+        ],
+        rows: table.dig(:cookies),
+      } %>
+    </div>
+  <% end %>
 
+  <div class="cookie-settings__form-wrapper govuk-!-padding-top-6">
```

Comment:
We try not to use the `!` override classes if possible - could you add the padding to the `.cookie-settings__form-wrapper` class? You can add it as a Sass function
```scss
padding: govuk-spacing(6)
``` 

which will give the same amount of padding as the override class.